### ISSUE TITLE
networkctl lldp -l fails to not ellipsize output

### ISSUE THREAD

**Issue by cooperlees** [systemd/systemd#24627]
### systemd version the issue has been seen with

systemd-250.3-6.6.hs+fb.el8.x86_64

### Used distribution

CentOS Stream release 8

### Linux kernel version used

5.12.0

### CPU architectures issue was seen on

x86_64

### Component

networkctl

### Expected behaviour you didn't see

I'd like to see the full "SYSTEM NAME".

### Unexpected behaviour you saw

Unable to get the full system name using -l (--full)

```console
[root@l33t ~]# networkctl lldp 
LINK             CHASSIS ID        SYSTEM NAME      CAPS        PORT ID           PORT DESCRIPTION
eth0             c2:18:50:40:d6:2c rsw069.p041.f01… ....r...... eth1/27/1         n/a

Capability Flags:
r - Router

1 neighbors listed.
[root@l33t ~]# networkctl lldp -l
LINK             CHASSIS ID        SYSTEM NAME      CAPS        PORT ID           PORT DESCRIPTION
eth0             c2:18:50:40:d6:2c rsw069.p041.f01… ....r...... eth1/27/1         n/a

Capability Flags:
r - Router

1 neighbors listed.
```

### Steps to reproduce the problem

- Have an lldp neighbor with a long hostname / system name.
  - It seems > 20 chars is enough
- Run `networkctl lldp --full` or `-l`

**Comment by said265** [comment]
>

**Comment by said265** [comment]
> >

**Comment by yuwata** [comment]
Should be already fixed by 27cd31c0b20ce09c1165ea2691ae5079b675d6d6. Closing.
